### PR TITLE
test: add basic cases for distributed TQL

### PR DESCRIPTION
**Pull request by waynexia**
I hereby agree to the terms of the [GreptimeDB CLA](https://gist.github.com/xtang/6378857777706e568c1949c7578592cc)

## What's changed and what's your intention?

Add two basic TQL queries. They will be run on both standalone and distributed modes.

## Checklist

- [x]  I have written the necessary rustdoc comments.
- [x]  I have added the necessary unit tests and integration tests.

### CODE REVIEW

File: tests/cases/standalone/common/tql/basic.result [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+TQL EVAL (0, 10, '5s') test{k="a"};
+
++-----+---------------------+---+
+| i   | j                   | k |
++-----+---------------------+---+
+| 2.0 | 1970-01-01T00:00:05 | a |
+| 2.0 | 1970-01-01T00:00:10 | a |
++-----+---------------------+---+
+
```
Comment (killme2008) [inline tests/cases/standalone/common/tql/basic.result:28]:
```suggestion
DROP TABLE test;
```

File: tests/cases/standalone/common/tql/basic.sql [hunk 2]
```diff
@@ -0,0 +1,10 @@
+CREATE TABLE test(i DOUBLE, j TIMESTAMP TIME INDEX, k STRING PRIMARY KEY);
+
+-- insert two points at 1ms and one point at 2ms
+INSERT INTO test VALUES (1, 1, "a"), (1, 1, "b"), (2, 2, "a");
+
+-- evaluate at 0s, 5s and 10s. No point at 0s.
+TQL EVAL (0, 10, '5s') test;
+
+-- the point at 1ms will be shadowed by the point at 2ms
+TQL EVAL (0, 10, '5s') test{k="a"};
```
Comment (killme2008) [inline tests/cases/standalone/common/tql/basic.sql:10]:
```suggestion
TQL EVAL (0, 10, '5s') test{k="a"};

DROP TABLE test;
```

**Review by MichaelScofield — APPROVED** [review]

File: tests/cases/standalone/common/tql/basic.sql (hunk 2 above)
Comment (waynexia) [reply tests/cases/standalone/common/tql/basic.sql:10]: good catch 👍

**Review by killme2008 — APPROVED** [review]
LGTM

**Pull request merged**